File: duplicate-assert/python-unittest/PkgImportRuleTest_testPkgImportRule.py
import unittest

class PkgImportRuleTest(unittest.TestCase):

    def test_pkg_import_rule(self):
        rule = PkgImportRule(True, False, "pkg", False, False)
        self.assertIsNotNone(rule, "Rule must not be null")
        self.assertEqual(AccessResult.UNKNOWN, rule.verify_import("asda"), "Invalid access result")
        self.assertEqual(AccessResult.UNKNOWN, rule.verify_import("p"), "Invalid access result")
        self.assertEqual(AccessResult.UNKNOWN, rule.verify_import("pkga"), "Invalid access result")
        self.assertEqual(AccessResult.ALLOWED, rule.verify_import("pkg.a"), "Invalid access result")
        self.assertEqual(AccessResult.ALLOWED, rule.verify_import("pkg.a.b"), "Invalid access result")
        self.assertEqual(AccessResult.UNKNOWN, rule.verify_import("pkg"), "Invalid access result")

File: duplicate-assert/CSharpXUnit/CSharpXUnit.Tests/Tests/PkgImportRuleTest_testPkgImportRule.cs
using Xunit;

public class PkgImportRuleTest
{
    [Fact]
    public void TestPkgImportRule()
    {
        var rule = new PkgImportRule(true, false, "pkg", false, false);
        Assert.NotNull(rule, "Rule must not be null");
        Assert.Equal(AccessResult.Unknown, rule.VerifyImport("asda"), "Invalid access result");
        Assert.Equal(AccessResult.Unknown, rule.VerifyImport("p"), "Invalid access result");
        Assert.Equal(AccessResult.Unknown, rule.VerifyImport("pkga"), "Invalid access result");
        Assert.Equal(AccessResult.Allowed, rule.VerifyImport("pkg.a"), "Invalid access result");
        Assert.Equal(AccessResult.Allowed, rule.VerifyImport("pkg.a.b"), "Invalid access result");
        Assert.Equal(AccessResult.Unknown, rule.VerifyImport("pkg"), "Invalid access result");
    }
}
